kymatio/scattering2d/backend/torch_skcuda_backend.py

```diff
@@ -162,34 +162,36 @@ def __call__(self, x):
 
         kernel = """
         extern "C"
-        __global__ void abs_complex_value(const ${Dtype} * x, ${Dtype}2 * z, int n)
+        __global__ void abs_complex_value(const ${Dtype} * x, ${Dtype} * z, int n)
         {
             int i = blockIdx.x * blockDim.x + threadIdx.x;
         if (i >= n)
             return;
-        z[i] = make_${Dtype}2(normf(2, x + 2*i), 0);
+        z[i] = normf(2, x + 2*i);
 
         }
         """
         fabs = _load_kernel('abs_complex_value', kernel, Dtype=_get_dtype(x))
-        fabs(grid=(self.GET_BLOCKS(int(out.nelement()) // 2), 1, 1),
+        fabs(grid=(self.GET_BLOCKS(int(out.nelement()) ), 1, 1),
```

Comment:
I still don't understand this change

Reply:
@edouardoyallon  gave me an explanation, something along the lines of "this is the number of entrys you are outputting". Before, it was  `self.GET_BLOCKS(int(out.nelement()) // 2` since we were out putting the real part, and the imaginary part could be 0.

Reply:
Basically, we are specifying the number of elements we are working with.